source/_patterns/00-atoms/components/info-bar.yaml

```diff
@@ -8,11 +8,15 @@ properties:
     type: string
     enum:
       - attention
+      - dismissible
       - info
       - multiple-versions
       - correction
       - success
       - warning
+  dismissibleOnOff:
+    type: boolean
+    default: false
```

Comment:
several properties like `cookieDuration` appear in `info-bar~*.json` examples but not in this schema?